package.json

```diff
@@ -22,7 +22,8 @@
     "start:relay": "yarn relay && concurrently --raw --kill-others 'yarn relay --watch' 'yarn start'",
     "sync-schema": "rm -rf data && mkdir data && graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
     "test": "sh scripts/test.sh",
-    "webpack": "webpack"
+    "webpack": "webpack",
+    "bundle-report": "NODE_ENV=production ANALYZE_BUNDLE=true webpack"
```

Comment:
Do you think it would make more sense to use an `argv` flag rather than an ENV var similar [to here](https://github.com/artsy/force/blob/master/package.json#L13)? Something like 
```
"bundle-report": "webpack --analyzeBundle"
```
And let the dev invoking the command pass in `NODE_ENV` (if need be) so that we can quickly see relative sizes without having to wait for full prod compile.

Reply:
Webpack tends to try to be really helpful about what arguments you pass to it and will fail when you pass an unknown argument to it. 

![image](https://user-images.githubusercontent.com/3087225/40890173-cd524cf8-673f-11e8-960a-201b008d3dad.png)

Generally though, even if it did work I'd still advice against that approach because it'd give people looking at it the impression that it was a built in feature of webpack rather than a configuration option being consumed by some code in the webpack build. I think using an ENV makes that very explicit.

As for the last bit, it's easy enough to run the bundle analyzer build manually to be able to give it whatever configuration it needs. I think the key here is that we want to do the right thing by default. A dev build may be very, very deceptive when it comes to looking at bundles. Things may be there that won't exist on prod and bundles could be wildly out of sync with that they'll look like in a prod build.